### ISSUE TITLE
The Download CSV button doesn't download a file

### ISSUE THREAD

**Issue by LandonSchropp** [philipmulcahy/azad#143]
Thanks for making such an awesome extension!

One thing I'm running into is the download CSV button doesn't actually download a file. In Chrome, it creates a new table within the browser, which can't be saved as a CSV. I'm not sure if this is how Chrome handles CSVs by default, but I think a better approach would be to force the file to download.

**Comment by philipmulcahy** [comment]
It is supposed to make a file that appears in your download folder. It can take a few seconds to appear. Please send me your console log if not.

Yours,

Philip

**Comment by kodetwo** [comment]
I have the same issue. Reinstalled the extension, Cleared cache, Tried on Chrome & Brave on Imac running Mojave & Macbook with High Sierra. Clicking the download button refreshes the page with a table but doesnt download a file.

**Comment by philipmulcahy** [comment]
@kodetwo and @LandonSchropp I can't do anything without a console log.

**Comment by kodetwo** [comment]
Hi Philip
Thanks for getting in touch. 
I read through a few open issues on your github page & came across 134 'Scrape wont complete'. I took the liberty of downloading the test version from there & followed your instructions & it worked perfectly  Version v1.9.17 was showing 50 errors from a list of 187 orders but the new version had zero errors. Taking a tip from someones post I deleted all cookies & restarted before giving it a try,  i don't know if that made a difference

Thank you for this excellent extension, it saved me a lot of time

Happy New Year!

Regards
Eddie
.
